### PR TITLE
.*: update changelog for v2.0.7

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 

add change log for v2.0.7

### CODE REVIEW

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Review by glorv — APPROVED** [review]

**Review by Ehco1996 — APPROVED** [review]

**Comment by lichunzhu** [comment]
/run-all-tests
https://ci.pingcap.net/blue/organizations/jenkins/dm_ghpr_new_test/detail/dm_ghpr_new_test/2677/pipeline

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+
+### Bug fixes
+
+- Fix the error that binlog event is purged when switching `enable-gtid` in source configuration from `false` to `true` [#2094](https://github.com/pingcap/dm/pull/2094)
+- Fix the memory leak problem of schema-tracker [#2133](https://github.com/pingcap/dm/pull/2133)
+
+### Improvements
+
+- Disable background statistic job in schema tracker to reduce CPU consumption [#2065](https://github.com/pingcap/dm/pull/2065)
+- Support regular expressions rules for Online DDL shadow and trash tables [#2139](https://github.com/pingcap/dm/pull/2139)
```
Comment (lichunzhu) [inline CHANGELOG.md:15]:
```suggestion
- Support regular expressions rules for online DDL shadow and trash tables [#2139](https://github.com/pingcap/dm/pull/2139)
```

**Comment by lichunzhu** [comment]
/run-all-tests
https://ci.pingcap.net/blue/organizations/jenkins/dm_ghpr_new_test/detail/dm_ghpr_new_test/2682/pipeline

**Pull request merged**